Android/CortexV2Example/README.md

```diff
@@ -0,0 +1,84 @@
+
+## Prerequisites
+- Android device (Android version >= 7)
+- EMOTIV App for Android
+- Emotiv headset (if you don't have, you can purchase one [here](https://www.emotiv.com))
+- Emotiv account (if you don't have, you can register [here](https://www.emotiv.com))
+
+## EMOTIV App for Android
+[What is EMOTIV App (for desktop)?](https://emotiv.gitbook.io/cortex-manual/)
+
+EMOTIV App for Android is required for any other apps which want to access EEG raw data and/or processed data from Emotiv headsets. You will need it to:
+* log in Emotiv system
+* manage headset connections
+* manage 3rd app permissions
+* It also provides a secure web socket connection (at wss://localhost:6868) for your app to read data from, make training action, etc. See more details in [Cortex API Documentation](https://emotiv.gitbook.io/cortex-api/).
+
+### Installing
+##### Unreleased version (only for Emotiv-partner developers)
+
+- You will receive an invitation email to a testing program for an unreleased version of the EMOTIV App.
+	- *Note: this is only available for our partner developers, not for everyone.*
+- You can click on the link provided in the email, follow the guide to install EMOTIV App on your Android device.
+	- You cannot install if you have any version of EMOTIV App installed on your device, so make sure you uninstall it (for all users) before installing a new version.
+	- If you send the link to others, they may not access to our unreleased version as long as they are not in our list.
+
+##### Released version
+
+Not available yet.
+
+## Try EMOTIV App
+- Open EMOTIV App, make sure you accept all permission request pop ups.
+- Log in using your Emotiv account. You are on the page of "Available Devices".
+- Turn on your Emotiv headset, wait for the headset to show up.
+	- *Note: If you don't see the headset after 30s, you can try to turn it off then on again.*
+- After the headset is shown up on the list, click to connect.
```

Comment:
do we need to note users that our system just supports to connect to only one headset at a moment?